src/Worker/Core/DurableTaskWorkerOptions.cs

```diff
@@ -145,6 +145,11 @@ public DataConverter DataConverter
     /// </summary>
     public bool IsVersioningSet { get; internal set; }
 
+    /// <summary>
+    /// Gets or sets a callback function that determines whether an orchestration should be accepted for work.
+    /// </summary>
+    public IOrchestrationFilter? OrchestrationFilter { get; set; }
```

Comment:
This needs `[Obsolete("Experimental")]` as well.

Reply:
Done.